### ISSUE TITLE
Please remove github stars from UI

### ISSUE THREAD

**Issue by 4n70w4** [getinsomnia/insomnia#5056]
### Expected Behavior

No github stars in UI

### Actual Behavior

![image](https://user-images.githubusercontent.com/38257723/183383921-7ee16df5-9aed-4d05-9193-8ec8e01797e3.png)

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

It interferes with work. It's distracting. There are misclicks.

### Insomnia Version

2022.5.0

### What operating system are you using?

Windows

### Operating System Version

Windows 10

### Installation method

download

**Comment by KristobalJunta** [comment]
I would +1 this. While it is cool to show what Insomnia has achieved - as it is indeed a really good tool - I find that having Github stars in the app's toolbar doesn't meaningfully add to my experience with the app. Neither does it add useful features for daily work, nor for esthetics - and is actually visually cluttering.

What I would suggest here is adding ability to hide the button via settings.

**Comment by sgarner** [comment]
I'm happy to give the repo a star, but I have to agree that putting this where the project selection menu used to be was a poor UX decision. It really gets in the way.

It might be better to occasionally present a "Give us a star on GitHub" dialog in the same way many apps have a "Leave us a review on the app store"-type nag screen.

**Comment by kikoosvk** [comment]
remove this please

**Comment by filfreire** [comment]
Heads up @subnetmarco and @wongstein, please see this issue and feedback from community.

**Comment by subnetmarco** [comment]
Hello everyone, an increasing number of GitHub stars helps the Insomnia application by bringing more visibility to the project, which in turn increases adoption and the amount of investment that we can (and will) allocate to the project. As we all know, open source is free to use, but it is not free to build: GitHub stars are one of the most inexpensive ways to generate an outsized outcome in the community by leveraging the tailwinds of increased adoption.

With that said, we may introduce the capability in the settings to potentially disable the button for users who don't want to see it in the header bar, or perhaps automatically do that for users that are subscribed to a paid plan.

In conclusion, GitHub stars are not just a vanity metric, they actually really help the project, that I am sure we all want to see get better over time.

**Comment by tjagithub** [comment]
If I could suggest having it match the overall design?

Perhaps move it to the starting page of Insomnia, and the projects page or a button by the user/settings buttons with just a ★/☆ icon?

**Comment by nicjay** [comment]
I don't mind it appearing by default, but I've long since starred the project. It serves no purpose for me and I'd like an option to hide it.

**Comment by prookie** [comment]
Please at least implement a setting to hide it for paying customers.

**Comment by subnetmarco** [comment]
@prookie yes, this is coming. We will make an update in such a way that paying customers won't see the star button.

**Comment by kevinfiol** [comment]
I agree that it doesn't enhance the experience at all, and clicking it to star the project doesn't make it go away, so I'm puzzled by this idea that it somehow benefits the project. Just seems like a random marketing idea that was done "because we can". I believe only allowing it to be removed for paid users is against the spirit of open-source and moves Insomnia more in the direction of a "freemium" model seen more commonly in closed-source software. I could understand paywalling cloud & sync features, but paywalling something as arbitrary and unhelpful (to the user) as a Github Stars counter is baffling. And also, by paywalling it, you are essentially admitting that the UI element *is* superfluous because you don't want users to be able to turn it off.

Just my 2 cents.

**Comment by ic3guy** [comment]
It's the mis-clicks. While I sympathize on what @subnetmarco has mentioned above, it's annoying. At a minimum, move it somewhere else. Maybe to the right side by the settings/account icons. It's incredibly distracting.

**Comment by tjagithub** [comment]
You can see that the Nightingale REST app has implemented this as a star icon: https://github.com/jenius-apps/nightingale-core

It looks much cleaner.

I still think rating/star begging comes off badly. It reminds me of _those_ mobile apps.

The haphazard way it was just thrown into the UI is concerning. Was there a particular pull request associated with this? How was it reviewed? Was it tested? Etc.

**Comment by jmoniatte** [comment]
Agree with the comments here, it should be removed or at least moved to a place where it does not break the UI.

Also surprised that this made it given the obvious attention given to details everywhere else in the product.

**Comment by subnetmarco** [comment]
When logging into the application with any subscription, including the Free subscription, the GitHub start button will disappear after the next version of Insomnia will be released. Later this year there will be a UI redesign on the header bar that will move the "projects dropdown" in a different location, therefore avoiding potential mis-clicks on the GH star button for logged-out users.

**Comment by reformed** [comment]
Why can't you just move it to the bottom right corner, next to the "Made with..." notice?

**Comment by jmoniatte** [comment]
It seems that the Github stars are back. Why ?? Please consider what most users have said: they are disruptive and add absolutely no value.

**Comment by subnetmarco** [comment]
For those that may not know this, GitHub stars disappear after logging into Insomnia with an account.

Furthermore, after the next GA (to be released this or next week), we are introducing a new GUI for the header bar of Insomnia. Among the many improvements, the GitHub stars won't be on the execution path when changing workspace/project anymore, therefore removing the risk of mis-clicks. Either way both with the new GUI, and today even, the GitHub stars already disappear after logging in with an Insomnia account (including a Free tier account).

Below screenshots of the new GUI in both logged and unlogged status:

Unlogged status:

<img width="1082" alt="Screen Shot 2022-12-14 at 2 33 33 PM" src="https://user-images.githubusercontent.com/964813/207609126-176e2f50-6425-4c0c-b310-b9eaae7947a2.png">

Logged status:

<img width="1085" alt="Screen Shot 2022-12-14 at 2 33 22 PM" src="https://user-images.githubusercontent.com/964813/207609211-dbe11059-9dfb-4782-a58a-41934ce5c157.png">

The GitHub stars in general are useful to the project as described in [this comment](https://github.com/Kong/insomnia/issues/5056#issuecomment-1212372179).

**Comment by reformed** [comment]
But this doesn't resolve the problem. If the stars go away for logged-in users, why do they need to be in such a distracting place for those who are not logged in? Again, why can't the stars be moved to the bottom right corner?

And it introduces another problem. Why should one have to log in? What if I don't want to create an Insomnia account? I am perfectly fine seeing the stars in a place that doesn't accidentally get clicked, like the bottom right corner.

**Comment by robclancy** [comment]
The stupid decision to do this triggered the original insomnia creator to create a much better alternative. You created competition with your stupid marketing bs.

**Comment by ic3guy** [comment]
I just got pinged by this comment. Thanks @robclancy for posting. I really like what I see with yaak.app. I haven't been working as a dev for the last two years, but pre-acquisition I was a super insomnia booster. But I do remember being pissed off by the stars in the UI and is probably why I subscribed to this github issue. Good on Gregory Schier jumping back into the API game. He clearly states on the front page of yaak.

"I was done building API clients after moving on from Insomnia in 2020, but the market shifted to mandatory accounts and other lock-in tactics.

So I started from scratch and built the tool I'm proud to put my name behind."

and

"Yaak is an **offline** and Git friendly app for HTTP, GraphQL, WebSockets, SSE, and gRPC."

The caveat is that I haven't used yaak, but those statements along with Schier's background would be enough for me to give it a try on a new project.

**Comment by subnetmarco** [comment]
I am not really sure what any of this has to do with Insomnia? Stop spamming and keep the conversation on topic.

**Comment by reformed** [comment]
How ironic @subnetmarco—if anything is spammy, it's Insomnia. I for one am grateful for these responses (which are totally relevant IMO).

**Comment by jmoniatte** [comment]
Totally agree, those are helpful comments, I was not aware of Yaak.

I'd like to also mention [bruno](https://www.usebruno.com/) as an awesome alternative that does not require accounts, and makes it easy to collaborate using git. Their [manifesto](https://docs.usebruno.com/introduction/manifesto) says it best.

**Comment by subnetmarco** [comment]
Removing spam and locking conversation.
